html/comparison.html.haml

```diff
@@ -15,6 +15,20 @@
     %script{:src => "assets/transition.js"}
     %script{:src => "assets/collapse.js"}
 
+    %script#alert{:type => "text/ng-template"}
+      #alert_container.scope{"ng-show" => "diff.meta.uninspected"}
+        .row
+          .col-xs-1
+            %span.dismiss{:title => "Collapse/Expand"}
+        .row
+          .row
+            .col-xs-1
+            .col-xs-7
+              Couldn't compare <strong>'{{diff.meta.uninspected[diff.meta.description_a].join(", ")}}'</strong> because they are not present in description <strong>'{{diff.meta.description_a}}'</strong>
+          .row
+            .col-xs-1
+            .col-xs-7
+              Couldn't compare <strong>'{{diff.meta.uninspected[diff.meta.description_b].join(", ")}}'</strong> because they are not present in description <strong>'{{diff.meta.description_b}}'</strong>
```

Comment:
The two sections should only be shown if there is something to, otherwise you can end up with something like this:

![image](https://cloud.githubusercontent.com/assets/203813/8614335/f8b3b264-26e8-11e5-9eed-5d63d1be7424.png)